src/components/AcceleratorReports/ChallengesMitigationBox.tsx

```diff
@@ -133,6 +133,10 @@ const ChallengesMitigationBox = (props: ReportBoxProps) => {
   const updateReportResponse = useAppSelector(selectReviewAcceleratorReport(requestId));
   const snackbar = useSnackbar();
 
+  useEffect(() => {
+    setChallengeMitigations(report?.challenges || []);
```

Comment:
@ice1080 this is similar to line 149. But without this funder report chllanges are not updated correctly, because getNonEmptyChallenges are not updated. Maybe you have a better idea on how to fix this?

Reply:
Maybe we can remove that useEffect and just use this one?

Reply:
Hmm ya I'm not sure the best way to do this. The problem is line 170, where modifying `challengeMitigations` calls `onChange` which modifies `report?.challenges`, causing an infinite loop (only in participant edit view)